extra/1-radio-stations.js

```diff
@@ -25,6 +30,17 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
+const getStations = () => {
```

Comment:
🔥 
Good use of invoking a function within a function, but maybe in this case, it should be used less often?
Since getAllFrequencies returns an array, do you think it would be beneficial to call it once, store its results in a local variable inside getStations, and use it wherever its needed? What do you think are the benefits of taking such an approach?